### PR TITLE
The code in constuct_setups.py can not generator setup.cfg correctly

### LINKED ISSUE #1325: CI bug: construct_setups.py have problem to generator setup.cfg file
#### Description
The code in constuct_setups.py file can not generator setup.cfg correctly, when the content of setup_requirements.txt likes below:
```
f5-sdk==3.0.11
six>=1.9.0
pyasn1>=0.1.9
pyasn1-modules>=0.0.8
```

Run command:
`python construct_setups.py --working_directory=/root/test/f5lbaas-agent --operating_system=redhat --version=1.1`

The result in setup.cfg is:
```
[bdist_rpm]
requires = f5-sdk< 3.0.12
    f5-sdk>= 3.0.11six>=1.9.0
    pyasn1>=0.1.9
    pyasn1-modules>=0.0.8
```

The expected result is:
```
[bdist_rpm]
requires = f5-sdk<3.0.12
    f5-sdk>=3.0.11
    six>=1.9.0
    pyasn1>=0.1.9
    pyasn1-modules>=0.0.8
```

@kaustriaf5

### PR DESCRIPTION
**Pull request by zhang-shengping**
For example:
The content of setup_requirements.txt likes below:
    f5-sdk==3.0.11
    six>=1.9.0
    pyasn1>=0.1.9
    pyasn1-modules>=0.0.8

Run command:
python construct_setups.py --working_directory=/root/test/f5lbaas-agent \
                           --operating_system=redhat \
                           --version=7

The result in setup.cfg is:
    [bdist_rpm]
    requires = f5-sdk< 3.0.12
    f5-sdk>= 3.0.11six>=1.9.0
    pyasn1>=0.1.9
    pyasn1-modules>=0.0.8

Change some codes in constuct_setups.py to adjust the output setup.cfg.

@kaustriaf5 
@zhaoqin-github 

#### What issues does this address?
Fixes #1325

### CODE REVIEW

File: f5-openstack-agent-dist/scripts/construct_setups.py [hunk 1]
```diff
@@ -287,7 +288,7 @@ def get_args():
 interprets their meaning before returning such content.
     """
     expected = ['working_directory', 'operating_system', 'version']
-    possible = ['json'].extend(expected)
+    possible = ['json'] + expected
```
Comment (kaustriaf5) [inline f5-openstack-agent-dist/scripts/construct_setups.py]: Was there a particular reason for this change?

**Comment by kaustriaf5** [comment]
The build log shows that the resulting setup.cfg is different from the expected output noted in the description: 
https://travis-ci.org/F5Networks/f5-openstack-agent/builds/422527828#L681

File: f5-openstack-agent-dist/scripts/construct_setups.py (hunk 1 above)
Comment (zhang-shengping) [reply f5-openstack-agent-dist/scripts/construct_setups.py]: ['json'].extend(expected) has no return value, variable "possible" is always None.

Comment (zhang-shengping) [reply f5-openstack-agent-dist/scripts/construct_setups.py]: If we do not use construct_setup.py as a command line for the test, it should not be an issue.

**Comment by zhang-shengping** [comment]
@kaustriaf5  yes, it is different. The description is an example since this patch will not add any dependencies. 

If you write the setup_requirements as the description and run command either `ython f5-openstack-agent-dist/scripts/universal_truth.py` or `python construct_setups.py --working_directory=/root/test/f5lbaas-agent --operating_system=redhat --version=1.1`. you will see the setup.cfg problem.

File: f5-openstack-agent-dist/scripts/construct_setups.py [hunk 2]
```diff
@@ -132,19 +132,20 @@ def _build_stdeb_cfg(wkg):
 
 def handle_equals(fh, fmt, name, specifier, pkg_type='rpm'):
     version_breakout = re.compile('(\d+)\.(\d+)\.(\d+)')
-    specifier_fmt = "{} {}.{}.{}"
+    specifier_fmt = "{}{}.{}.{}"
```
Comment (zhaoqin-github) [inline f5-openstack-agent-dist/scripts/construct_setups.py]: Do not need to change this line.

File: f5-openstack-agent-dist/scripts/construct_setups.py [hunk 3]
```diff
@@ -132,19 +132,20 @@ def _build_stdeb_cfg(wkg):
...
     if '-' in specifier:
         specifier = re.sub('-\d+', '', specifier)
     match = version_breakout.search(specifier)
     assert match, "{} could not be matched with {}".format(
         specifier, version_breakout.pattern)
     major, minor, patch = match.groups()
-    lt_specifier = '<<' if pkg_type == 'deb' else '<'
+    lt_specifier = '<<' if pkg_type == 'deb' else '< '
```
Comment (zhaoqin-github) [inline f5-openstack-agent-dist/scripts/construct_setups.py]: Do not need to change this line

File: f5-openstack-agent-dist/scripts/construct_setups.py [hunk 4]
```diff
@@ -132,19 +132,20 @@ def _build_stdeb_cfg(wkg):
...
     fh.write(fmt % (name,
                     specifier_fmt.format(
                         lt_specifier, major, minor, int(patch) + 1)))
     fh.write('\n    ')
-    fh.write(fmt % (name, specifier_fmt.format(">=", major, minor, patch)))
+    fh.write(fmt % (name, specifier_fmt.format(">= ", major, minor, patch)))
```
Comment (zhaoqin-github) [inline f5-openstack-agent-dist/scripts/construct_setups.py]: Do not need to change this line.

**Pull request merged**